apps/emqx/src/emqx_persistent_message_ds_replayer.erl

```diff
@@ -123,7 +123,7 @@ commit_offset(
 
 -spec poll(emqx_persistent_session_ds:id(), inflight(), pos_integer()) ->
     {emqx_session:replies(), inflight()}.
-poll(SessionId, Inflight0, WindowSize) when WindowSize > 0, WindowSize < 16#7fff ->
+poll(SessionId, Inflight0, WindowSize) when WindowSize > 0, WindowSize =< 16#ffff ->
```

Comment:
Oh, I also addressed it in https://github.com/emqx/emqx/pull/12029/files#diff-3d714666e0c828c1fd3b7fd7884623dcf544f641be03e0b0a46153b3ed5fa277R169.